demo/e2e/app.e2e-spec.ts

```diff
@@ -0,0 +1,14 @@
+import { DemoPage } from './app.po';
+
+describe('demo App', () => {
+  let page: DemoPage;
+
+  beforeEach(() => {
+    page = new DemoPage();
+  });
+
+  it('should display message saying app works', () => {
```

Comment:
Do we want to have e2e tests against our documentation @pkozlowski-opensource ? I can see some value in this, but it also has cost too.